### DISCUSSION TITLE
Multi Animation in Cinema 4D

### DISCUSSION THREAD

**Discussion by Olega95** [google/model-viewer#2217] [Q&A]
Good day! Please tell me. You have multi-animation models in one file. How do you do it? What program? And is it possible to do this through Cinema 4D? I searched everything I could and found nothing. Help me please!

**Comment by elalish** [comment]
I don't know anything about Cinema 4D; you can make animated glTF models in lots of programs, like Blender, Maya, etc. You'll have to ask on their forums.

**Comment by Olega95** [comment]
I'm sorry, maybe I didn't explain it correctly. I know how to animate a model. The question is how to do multi-animation in one gltf / glb file. That is, there are several animations in one model. I am interested to know how you, as developers, made multiple animations in one model.

**Comment by elalish** [comment]
I didn't actually make any of the animated models on modelviewer.dev. But multiple animations in a glTF is quite common and model-viewer supports a short cross-fade when switching animations to smooth the transition.

**Comment by AIRS-EXR** [comment]
> I didn't actually make any of the animated models on modelviewer.dev. But multiple animations in a glTF is quite common and model-viewer supports a short cross-fade when switching animations to smooth the transition.
Please tell me how I can achieve switching animations by using model-viewer?how to do it?

**Comment by elalish** [comment]
@AIRS-EXR Look at the third example of animations on modelviewer.dev: https://modelviewer.dev/examples/animation.html

**Comment by KibbleKode** [comment]
@AIRS-EXR did you ever figure this out? i'm in the same situation now. Ive added the animation as motions clips but they do not show. i've tried adding them as separate 'takes' too but that doesn't work.
